packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -704,6 +776,17 @@ class TransformLayer extends OffsetLayer {
     builder.pop();
   }
 
+  @override
+  S find<S>(Offset regionOffset) {
+    if (_invertedTransform == null) {
+      _invertedTransform ??= new Matrix4.zero();
```

Comment:
no need for `??`, since you know it's null.

Reply:
Fixed